src/progress_bar.rs

```diff
@@ -122,7 +122,7 @@ impl ProgressBar {
     /// [`ProgressBarIter`] completes and
     /// [`ProgressBar::is_finished()`] is false.
     /// If you don't want the progress bar to be automatically finished then
-    /// call `on_finish(None)`.
+    /// call `on_finish(None)`. TODO: Is this method even public?
```

Comment:
What does on_finish refer to?

Reply:
I think it's now called `with_finish()` in the public API.